src/translations/vnote_zh_CN.ts

```diff
@@ -9,42 +9,42 @@
         <translation>所有幻词：</translation>
     </message>
     <message>
-        <location filename="../veditor.cpp" line="392"/>
+        <location filename="../veditor.cpp" line="441"/>
```

Comment:
Do not need to change `vnote_zh_CN.qm` and `vnote_zh_CN.ts`. I will change them together before a release.

Reply:
Yes. Changing the translation files in every commit makes that commit huge, which is hard to track.